MatrixSDK/Contrib/Swift/MXRestClient.swift

```diff
@@ -1912,6 +1912,7 @@ public extension MXRestClient {
                                                relationType: String?,
                                                eventType: String?,
                                                from: String?,
+                                               direction: MXTimelineDirection,
```

Comment:
You should probably update the docs too.

Reply:
🤦‍♂️